flycheck.el

```diff
@@ -7777,6 +7763,32 @@ See https://github.com/processing/processing/wiki/Command-Line"
   ;; This syntax checker needs a file name
   :predicate (lambda () (buffer-file-name)))
 
+(flycheck-define-checker pug
+  "A Pug syntax checker using the pug compiler.
+
+See URL `https://pugjs.org/'."
+  :command ("pug" "-p" (eval (expand-file-name (buffer-file-name))))
+  :standard-input t
+  :error-patterns
+  ;; errors with includes/extends (e.g., missing files)
+  ((error "Error: " (message) (zero-or-more not-newline) "\n"
+          (zero-or-more not-newline) "at "
+          (zero-or-more not-newline) " line " line)
+   ;; syntax/runtime errors(e.g, type errors)
```

Comment:
There's a space missing before the opening parenthesis, and the comma after `e.g` should be a dot :)